contributing/ci-bio-formats.txt

```diff
@@ -14,6 +14,10 @@ All jobs are listed under the :jenkinsview:`Bio-Formats` view tab of Jenkins.
 		* :term:`BIOFORMATS-5.1-latest`
 		*
 
+	-	* Generates the latest generated-sources
+		* :term:`BIOFORMATS-5.1-latest-generated-sources`
```

Comment:
Not sure what this job is for.  There no delta once it's merged!  What's the baseline for comparison here?